inscriptions/ISic030010.xml

```diff
@@ -109,7 +115,7 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0500" notAfter-custom="-0450">First half V cent. BCE</origDate> <!-- check date -->
                         </origin>
                         <provenance type="found">Found in the archaeological area of Selinunte<geo>37.5853775, 12.8298666</geo>.</provenance> <!-- check provenance -->
```

Comment:
It was found in the necropolis in Contrada Buffa. Please leave the generic coordinates for Selinunte for now and add a comment to flag it out